tests/security/apparmor/aa_logprof.pm

```diff
@@ -66,27 +77,28 @@ sub run {
     # Upload audit.log for reference
     upload_logs "$log_file";
 
-    script_run_interactive(
-        "aa-logprof -d $aa_tmp_prof",
-        [
-            {
-                prompt => qr/\(A\)llow/m,
-                key    => 'a',
-            },
-            {
-                prompt => qr/\(S\)ave Changes/m,
-                key    => 's',
-            },
-        ],
-        30
-    );
+    script_run_interactive("aa-logprof -d $aa_tmp_prof", $interactive_str, 30);
 
     foreach my $item (@aa_logprof_items) {
         validate_script_output "cat $aa_tmp_prof/usr.sbin.nscd", sub { m/$item/ };
     }
 
     $self->aa_tmp_prof_verify("$aa_tmp_prof", 'nscd');
     $self->aa_tmp_prof_clean("$aa_tmp_prof");
+
+    if (is_tumbleweed || is_sle('>15-sp2')) {
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check, e.g. use `if (!is_sle('<15-sp1') && !is_leap('<15.1'))`

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
> Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check, e.g. use `if (!is_sle('<15-sp1') && !is_leap('<15.1'))`
> 
> See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Sure (though some times the policies makes coding not very convenient).